src/app/shared/request/request.service.ts

```diff
@@ -96,7 +96,7 @@ export class RequestService {
   }
 
   private getEndpointUrl(endpoint, isLoginAPI?: boolean) {
-    let endpointUrl = this.prefixUrl + endpoint;
+    let endpointUrl = ( this.storage.stackConfig ? this.storage.stackConfig.coreApi : this.prefixUrl ) + endpoint;
```

Comment:
there is no `this.prefixUrl` anymore, it should be removed from here an the app module as well.
If there is no stack url, we should throw error.